themes/socialblue/templates/node/event/node--event--full--sky.html.twig

```diff
@@ -1,2 +1,6 @@
- {# This file overrides node--event--full.html.twig #}
+{# This file overrides node--event--full.html.twig #}
 {% extends "node--full--sky.html.twig" %}
+
+{% block nodefull_body %}
+  {{ content|without('flag_follow_content', 'like_and_dislike', 'field_event_location', 'field_event_address', 'enrollments_count', 'links') }}
```

Comment:
Could we add a comment here that these are displayed in the hero instead? This way if that changes in the future we know quicker why this was done :)

Reply:
Done

Reply:
@Kingdutch, @ronaldtebrake I have checked that address fields was fixed recently by @bramtenhove in a commit: 89b1ae0. But event enrollments and event status still displaying there (see the screenshot from normal heme).
We can't hide those fields in the config, since they are added `social_event_node_view_alter`, so I think we just need to hide them here as well. 
Please check again.